blueman/gui/applet/PluginDialog.py

```diff
@@ -159,45 +169,97 @@ def __init__(self, applet: "BluemanApplet") -> None:
         self.sig_b: int = self.applet.Plugins.connect("plugin-unloaded", self.plugin_state_changed, False)
         self.connect("delete-event", self._on_close)
 
-        self.list.set_cursor(Gtk.TreePath.new_first())
-
         close_action = Gio.SimpleAction.new("close", None)
         close_action.connect("activate", lambda x, y: self.close())
+
         self.add_action(close_action)
 
-    def list_compare_func(self, _treemodel: Gtk.TreeModel, iter1: Gtk.TreeIter, iter2: Gtk.TreeIter, _user_data: object
-                          ) -> int:
-        a = self.list.get(iter1, "activatable", "name")
-        b = self.list.get(iter2, "activatable", "name")
+    def _add_plugin_action(self, name: str, state: bool, activatable: bool) -> None:
+        logging.debug(f"adding action: {name}")
+        action = Gio.SimpleAction.new_stateful(
+            name, None, GLib.Variant.new_boolean(False)
+        )
+        action.set_property("enabled", activatable)
+        action.set_state(GLib.Variant.new_boolean(state))
```

Comment:
Why not initialize with `state` right away?

Reply:
Don't think there is a reason. I'll double check to be sure and update the commit.

Reply:
Fixed.